src/test/ui/extern/extern-types-not-sync-send.stderr

```diff
@@ -5,6 +5,7 @@ LL |     assert_sync::<A>();
    |                   ^ `A` cannot be shared between threads safely
    |
    = help: the trait `Sync` is not implemented for `A`
+   = help: the trait `Sync` is implemented for `any::OpaqueStrBytes`
```

Comment:
What is happening here? That error messages seems weird

Reply:
This is diagnostic logic not ignoring private `core` types heh.